src/service/Notes/updateNoteService.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+const updateNoteService = async (currentNote, tokenValue) => {
+  try {
+    const response = await axios.post(
+      `/api/notes/${currentNote._id}`,
+      { note: { ...currentNote } },
+      {
+        headers: {
+          authorization: tokenValue,
+        },
+      }
+    );
+    return response;
+  } catch (error) {
+    console.log(error);
+    error;
+  }
+};
+
+const updateNotePinService = async (currentNote, tokenValue) => {
+  try {
+    const response = await axios.post(
+      `/api/notes/${currentNote._id}`,
+      { note: { ...currentNote, isPinned: !currentNote.isPinned } },
```

Comment:
this `isPinned` you can change before calling this service...which will reduce the repeated code. 
only the `updateNoteService` will do the job.

Reply:
This updatepinservice is specific to cards when they are saved, the general updateNoteservice is for editor